andela/rogue-backend
# sends an email to users for password reset

#### What does this PR do?
* Sends an email to a user who has forgotten her/his password for password reset.

#### Description of Task to be completed?
* A user should be able to reset his/her whenever they forget.
* On the login page, the user clicks the **forgotpassword?** link and an email is sent to his email address.

#### How should this be manually tested?
* Clone the repo and `cd` into the project directory
* Run `yarn install` to install the project's dependency
* Go to `\src\seeders\20190821163231-demo-user.js` and replace the email address in the file with your email address.
* Run `yarn migrate` to create the database's tables.
* Run `yarn seed` to fill the database with sample data.
* Run `yarn build` to compile the code with `babel`
* Run `yarn start`
* Open postman on your machine, enter this url: `http://localhost:3000/resetpassword` and `send a post request`
* Check your inbox for the `reset password` button
#### What are the relevant pivotal tracker stories?
[#PT-Story-ID](http://PT-Story-Link)
https://www.pivotaltracker.com/n/projects/2381976/stories/167727630

#### Any background context you want to add?
